packages/icons/src/_icons.scss

```diff
@@ -1,3 +1,4 @@
 .lucca-icon {
 	@extend %lucca-icon;
+	@include buildIconClasses;
```

Comment:
Here the selector `.lucca-icon` is repeated for each icon in the final dist, and we have ~200 icons, wich leads to somewhat wasted dist size. Could be optimized, but it's a rather marginal gain